llvm/test/CodeGen/RISCV/rv64zbc-zbkc-intrinsic.ll

```diff
@@ -4,7 +4,7 @@
 ; RUN: llc -mtriple=riscv64 -mattr=+zbkc -verify-machineinstrs < %s \
 ; RUN:   | FileCheck %s -check-prefix=RV64ZBC-ZBKC
 
-declare i64 @llvm.riscv.clmul.i64(i64 %a, i64 %b)
```

Comment:
You can't remove test coverage for `llvmriscv.clmul.i64`

Reply:
This was one of the parts I'm very unsure about: Should we remove `llvm.riscv.clmul.i64` (or make it an alias somehow?) Having the `riscv` target specific version of this intrinsic seems not great, but I don't know what the best approach for maintaining simplicity and backward compatibility is.

Reply:
We can remove llvm.riscv.clmul, but we need to update clang to not use it, and we need to recognize it in lib/IR/AutoUpgrade.cpp and remap it to the new intrinsic.

None of that should be in the patch that introduces the new intrinsic though.